src/particles/beam/BeamParticleContainerInit.cpp

```diff
@@ -1051,7 +1090,9 @@ InitBeamFromFile (const std::string input_file,
     auto& particle_tile = getBeamInitSlice();
     auto old_size = particle_tile.size();
     auto new_size = old_size + num_to_add;
+    particle_tile.define(3, 0);
```

Comment:
```suggestion
    if (m_do_spin_tracking) {
        particle_tile.define(3, 0);
    }
```

Reply:
Somewhere before the resize, there should also be a setArena for the three spin components, like in BeamParticleContainer.cpp line 95 to 104.